File: OPENAPI-REST-API/openapi-client/javascript-closure-angular/API/Client/Bandwidth.js
goog.provide('API.Client.Bandwidth');

/**
 * A bandwidth option available for a dedicated server configuration.
 * @record
 */
API.Client.Bandwidth = function() {}

/**
 * @type {!number}
 * @export
 */
API.Client.Bandwidth.prototype.id;

/**
 * @type {!string}
 * @export
 */
API.Client.Bandwidth.prototype.shortDesc;

/**
 * @type {!number}
 * @export
 */
API.Client.Bandwidth.prototype.monthlyPrice;

/**
 * @type {!string}
 * @export
 */
API.Client.Bandwidth.prototype.monthlyPriceDisplay;

File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/Bandwidth.scala
package io.swagger.server.model

import java.math.BigDecimal

/**
 * A bandwidth option available for a dedicated server configuration.
 *
 * @param id 
 * @param short_desc 
 * @param monthly_price 
 * @param monthly_price_display 
 */
case class Bandwidth (
  id: Option[Int],
  short_desc: Option[String],
  monthly_price: Option[BigDecimal],
  monthly_price_display: Option[String]
)

